docs/root/operations/admin.rst

```diff
@@ -190,7 +190,48 @@ The fields are:
 
   .. http:get:: /stats?format=json
 
-  Outputs /stats in JSON format. This can be used for programmatic access of stats.
+  Outputs /stats in JSON format. This can be used for programmatic access of stats. Counters and Gauges
+  will be in the form of a set of (name,value) pairs. Histograms will be under the element "histograms",
+  that contains "supported_quantiles" which lists the quantiles supported and an array of computed_quantiles
+  that has the computed quantile for each histogram.
+
+  The JSON structure would as shown below.
+
+  .. code-block:: json
+
+  {
+    "stats": [
+        {
+            "name": "example.counter",
+            "value": 0
+        },
+        {
+            "name": "example.gauge",
+            "value": 0
+        },
+        {
+            "histograms": {
+                "supported_quantiles": [...],
+                "computed_quantiles": [
+                    {
+                        "name": "example.histogram",
+                        "values": [
+                            {
+                                "interval": null,
```

Comment:
As above, it would be nice to see an example of a histogram with non-null quantiles values since that will be the common case.  But it'd also be nice to keep some sort of `null` example so the user knows what a no-data hist will look like.  

Side note: will this ever print histograms that have no samples in both cumulative and internal since that would mean its `used()` method would return `false`?

Reply:
I will make those changes. and regarding used, it currently prints all histograms. And I think it is a good point, it would be good filter the unused ones. I will make that change as well.